src/components/[guild]/claim-poap/components/MintLinksModal.tsx

```diff
@@ -0,0 +1,67 @@
+import { useAccount } from "wagmi"
+import { Modal } from "../../../common/Modal"
+import {
+  Box,
+  Center,
+  HStack,
+  Icon,
+  Link as LinkButton,
+  ModalBody,
+  ModalCloseButton,
+  ModalContent,
+  ModalHeader,
+  ModalOverlay,
+  Spinner,
+  Text,
+} from "@chakra-ui/react"
+import { ArrowSquareOut, CheckCircle } from "phosphor-react"
+import ErrorAlert from "../../../common/ErrorAlert"
+
+export const MintLinksModal = ({ isOpen, onClose, isLoading, error, response }) => {
```

Comment:
nit, but let's call it `MintLinkModal` (both the file and the component), because we only show 1 mint link in it